packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -540,28 +575,21 @@ export class TeamsBotImpl {
 
     await handler?.start(ProgressBarConstants.DEPLOY_STEP_START);
 
-    const packDir = this.config.scaffold.workingDir;
-
     await handler?.next(ProgressBarConstants.DEPLOY_STEP_NPM_INSTALL);
-
     await LanguageStrategy.localBuild(
       this.config.scaffold.programmingLanguage!,
-      packDir,
+      workingDir,
       this.config.deploy.unPackFlag === "true" ? true : false
     );
 
     await handler?.next(ProgressBarConstants.DEPLOY_STEP_ZIP_FOLDER);
-    const zipBuffer = utils.zipAFolder(packDir, DeployConfigs.UN_PACK_DIRS, [
+    const zipBuffer = utils.zipAFolder(workingDir, DeployConfigs.UN_PACK_DIRS, [
```

Comment:
it seems there're some unrelated changes, is it caused by git merge?

Reply:
No. It is just some tiny code refactoring and I change it in passing.

Reply:
The original `packDir` is just the same variable as the `workingDir`. They are just the same value and change the name. So I remove one of them.